packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md

```diff
@@ -0,0 +1,672 @@
+
+The user creation operation brings the user’s application profile from Okta over to the service provider. An user's application profile represents the key-value pairs defined in the Profile tab when a user is added.
+
+To enable user provisioning, an Okta administrator must configure the provisioning options under the Okta Admin Console. For more information on enabling the provisioning features of your SCIM application, see [Provisioning and Deprovisioning](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Provisioning_Deprovisioning_Overview), under the **Accessing Provisioning Features** section.
+
+After saving the application configuration, Okta runs two requests:
+
+    GET /scim/v1/users?startIndex=1&count=2 HTTP/1.1
+    User-Agent: Okta SCIM Client 1.0.0
+    Authorization: <Authorization credentials>
+
+and
+
+    GET /scim/v1/Groups?startIndex=1&count=100 HTTP/1.1
+    User-Agent: Okta SCIM Client 1.0.0
+    Authorization: <Authorization credentials>
+
+**Note:** The query parameters that Okta sends at this point are always constant.
+
+Next the Okta administrator checks the option for **Create Users** under the **Provisioning >> To App** tab. After this is complete, when a user is assigned to the application in Okta, the following requests are sent to the SCIM server:
+
+#### Determine if the user already exists
+
+<ApiOperation method="get" url="/users" />
+
+  Okta checks that the user exists on the SCIM server through a GET operation with the `userName` filter or any other filter that was configured with the SCIM application. This check is performed using the `eq` (equal) operator and is the only one necessary to successfully provision users with Okta.
+
+**Note:** The filter must identify a unique property of the user in the service provider profiles.
+
+For Okta Integration Network (OIN) applications, this filter is configured during the review process done by the Okta Application Analyst assigned to evaluate the application submission. Application submissions are processed through the [OIN Manager](https://oinmanager.okta.com).
+
+The requests from Okta to the service provider are of the form:
+
+    GET /scim/v1/users?filter=userName%20eq%20%22test.user%40okta.local%22&startIndex=1&count=100 HTTP/1.1
+    User-Agent: Okta SCIM Client 1.0.0
+    Authorization: <Authorization credentials>
+
+The SCIM application must check the filter provided and return and empty response if no users match the search criteria. For example:
+
+    HTTP/1.1 200 OK
+    Date: Fri, 18 Oct 2019 06:39:03 GMT
+    Content-Type: application/json
+
+    {
+        "schemas": ["urn:scim:schemas:core:1.0"],
+        "totalResults": 0,
+        "startIndex": 1,
+        "itemsPerPage": 0,
+        "Resources": []
+    }
+
+If the SCIM server does return a user, Okta automatically matches the result with the user in Okta, sending the user application profile to the SCIM server.
+
+<ApiOperation method="put" url="/users/{userId}" />
```

Comment:
Is there a heading missing here? This section normally appears under a heading describing the operation.

Reply:
I cleaned up the create user section to hopefully make it flow better.